sections/futures/UserInfo/OpenOrdersTable.tsx

```diff
@@ -112,12 +112,14 @@ const OpenOrdersTable: React.FC = () => {
 	}, [cancelNextPriceOrder.hash, executeNextPriceOrder.hash]);
 
 	const rowsData = useMemo(() => {
-		if (!cancelling) return openOrders;
-		const copyOrders = [...openOrders];
-		const cancellingIndex = copyOrders.findIndex((o) => o.id === cancelling);
-		copyOrders[cancellingIndex] = { ...copyOrders[cancellingIndex], isCancelling: true };
-		return copyOrders;
-	}, [openOrders, cancelling]);
+		const ordersWithCancel = openOrders.map((o) => ({ ...o, cancel: () => onCancel(o) }));
```

Comment:
@Tburm the cancel issue turned out to be a really strange one where the cell render method in the table was never getting the latest callback and so crossMarginAccountContract was always null, unless you changed tabs and back again. I tried a few things but could never get it to work as is, so I moved it into the table data itself like this.